openupgrade_scripts/scripts/google_calendar/16.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,12 @@
+---Models in module 'google_calendar'---
+---Fields in module 'google_calendar'---
+google_calendar / res.users                / google_cal_account_id (many2one): DEL relation: google.calendar.credentials
+google_calendar / res.users                / google_calendar_account_id (many2one): NEW relation: google.calendar.credentials
+# DONE pre-migration: rename google_cal_account_id -> google_calendar_account_id
+
+---XML records in module 'google_calendar'---
+ir.model.constraint: google_calendar.constraint_res_users_google_token_uniq (changed definition: is now 'unique(google_calendar_account_id)' ('unique(google_cal_account_id)'))
```

Comment:
this needs to be handled by deleting the constrain in pre. use delete_sql_constraint_safely()

Reply:
Ok, but could you explain to me why i need to this. please?

Reply:
If you dont drop the constraint, the definition will not be updated by the ORM. So your database will be inconsistant. If you drop the contraint, it will be recreated correctly.

Reply:
Thank you